### PR TITLE
Add Results class and accompanying methods

### PR DESCRIPTION
**Pull request by albonigrum**
Add Results class and accompanying methods
Tiny fixes

### CODE REVIEW

File: shlyapa/config.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 
 class Config(object):
-    def __init__(self, type=None, number_players=None, number_words=None, number_tours=3, is_last_turn_in_tour_divisible=True):
+    def __init__(self, type=None, number_players=None, number_words=None, number_tours=None, is_last_turn_in_tour_divisible=None):
```
Comment (vgrebenschikov) [inline shlyapa/config.py:8]: Why not default for number of tours?

File: shlyapa/shlyapa.py [hunk 2]
```diff
@@ -157,8 +228,32 @@ def print_shlyapa(game, end):
...
+    print("======Guessed Table======")
+    for i in range(g.config.number_tours):
+        print(gss_t[i])
+
+    t_s = g.get_total_score_results()
+    exp_s = g.get_explained_score_results()
+    gss_s = g.get_guessed_score_results()
+    print("#|    T|    E|    G")
+    for i in range(g.config.number_players):
+        print(i, t_s[i], exp_s[i], gss_s[i], sep="|    ")
```
Comment (vgrebenschikov) [inline shlyapa/shlyapa.py:259]: worth to use width formatter to make table nicer here

File: shlyapa/shlyapa.py [hunk 3]
```diff
@@ -40,8 +40,55 @@ def is_new(self):
         def is_end(self):
             return self.number_explained_words == self.number_words
 
+    class Results:
```
Comment (vgrebenschikov) [inline shlyapa/shlyapa.py:43]: Probably move this class to results.py

File: shlyapa/config.py (hunk 1 above)
Comment (albonigrum) [reply shlyapa/config.py:8]: For unifomity. I think explicit usage number_tours and is_last_turn_in_tour_divisible better

**Pull request merged**